rssalertbot

```diff
@@ -39,11 +44,12 @@ BOGUS_TIMEZONES = {
     'EDT': '-0400',
 }
 
-logging.basicConfig(level=logging.ERROR)
+logging.basicConfig(level=logging.ERROR,
+                    format='%(levelname)-7s [%(module)s.%(funcName)s:%(lineno)d] %(message)s')
 log = logging.getLogger()
 
 
-class Feed(object):
+class Feed():
```

Comment:
If 'yer changing things to be more hard core about python3, you can remove the () altogether.